src/ucp/api/ucp.h

```diff
@@ -2357,6 +2377,37 @@ ucs_status_t ucp_mem_advise(ucp_context_h context, ucp_mem_h memh,
                             ucp_mem_advise_params_t *params);
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief Memory copy routine
+ *
+ * This routine performs memory copy from @a src to @a dest buffer, given the
+ * corresponding data and memory types specified in @a params. Refer to
+ * @ref ucp_request_param_t.target_datatype for @a dest data type and
+ *  @ref ucp_request_param_t.datatype for @a src data type.
+ *
+ * @param [in]  worker      Worker handle
+ * @param [in]  dest        Destination buffer to copy into
+ * @param [in]  dest_count  Number of elements of
+ *                          @ref ucp_request_param_t.target_datatype to copy from
```

Comment:
can put `@ref ucp_request_param_t.target_datatype` in the previous line as for `src_count`

Reply:
the line would be too long then